src/main/ui/package.json

```diff
@@ -23,7 +23,8 @@
     "react": "^15.4.2",
     "react-bootstrap": "^0.30.7",
     "react-dom": "^15.4.2",
-    "react-router": "^3.0.2"
+    "react-router": "^3.0.2",
+    "react-select": "*"
```

Comment:
How did you install this? It should be showing a specific version. When I do `npm install --save react-select` it _does_ update this file to show `^0.9.1` instead of *